components/history/core/browser/sync/brave_typed_url_sync_bridge.cc

```diff
@@ -0,0 +1,34 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/history/core/browser/sync/brave_typed_url_sync_bridge.h"
+
+#include <memory>
+#include <utility>
+
+#include "brave/components/brave_sync/features.h"
+
+namespace syncer {
+class ModelTypeChangeProcessor;
+}
+
+namespace history {
+
+class HistoryBackend;
+class TypedURLSyncMetadataDatabase;
```

Comment:
I don't think you need these forwards.

Reply:
fixed at https://github.com/brave/brave-core/pull/16705/commits/875844cd3a0e02a8c06a780660fd45e1dd4bae3a